src/Raven.Server/Documents/Patch/BlittableObjectInstance.cs

```diff
@@ -541,6 +541,12 @@ public static JsValue GetJsValueForLazyNumber(Engine engine, LazyNumberValue val
             _doc = doc;
         }
 
+        public bool TryGetOriginalDocumentIfUnchanged(out Document doc)
+        {
+            doc = _doc;
+            return _doc != null && Changed == false;
```

Comment:
Shouldn't we out `null` if this condition is false?

Reply:
Yes, but given that we have to set _something_ there, and that the calling code uses the `bool`...